### PR TITLE
CLUS-firepower-restapit106-md1

### PR DESCRIPTION
**Pull request by ReneeBornstein**
* lenghtened sentence space
* added words to make whole
   sentences.

### CODE REVIEW

File: labs/firepower-restapi-106/1.md [hunk 1]
```diff
@@ -1,23 +1,24 @@
 # FMC - REST - API - 106
 
-## Lab Objective
+## Lab objective
```
Comment (paulzim) [inline labs/firepower-restapi-106/1.md]: Should just be "Objectives"

File: labs/firepower-restapi-106/1.md [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
 
-Basic understanding of FMC object(s) and CRUD operations using REST API.  
+This lab teaches a basic understanding of FMC object(s) and CRUD operations using REST API.  
 
-Completion time: 30 minutes
 
 ## Prerequisites
-Basic understanding of REST principles, Firewalls, FMC REST API
+It's best if you have a basic understanding of REST principles, Firewalls, and FMC REST API.
```
Comment (paulzim) [inline labs/firepower-restapi-106/1.md]: The original is better... "It's best..." is a judgement statement

**Review by paulzim — CHANGES_REQUESTED** [review]

File: labs/firepower-restapi-106/1.md [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
-Completion time: 30 minutes
 
 ## Prerequisites
-Basic understanding of REST principles, Firewalls, FMC REST API
+Basic understanding of REST principles, Firewalls, and FMC REST API.
 
-## Learning Objective
-You will learn how to do operations (GET, POST) using Firepower management center (FMC) objects to configure NGFW and other Firepower devices.
+
+## Learning objective
```
Comment (annegentle) [inline labs/firepower-restapi-106/1.md]: This section "Learning objective" is redundant and can be removed.

File: labs/firepower-restapi-106/1.md [hunk 4]
```diff
@@ -27,37 +28,27 @@ which you cannot edit, with the exception of security zones and interface groups
...
-domains.Subdomain users can add and delete interfaces from ancestor zones and groups, but cannot delete
+domains. Subdomain users can add and delete interfaces from ancestor zones and groups, but cannot delete
 or rename the zones/groups.* </mark>
 
 Object names must be unique within the domain hierarchy. The system may identify a conflict with the name
 of an object you cannot view in your current domain. For objects that support grouping, you can group objects in the current domain with objects inherited from ancestor domains.
 
-Object overrides allow you to define device-specific or domain-specific values for certain types of object,
-including network, port, VLAN tag, and URL.
+Object overrides allow you to define device-specific or domain-specific values for certain types of object, including network, port, VLAN tag, and URL.
```
Comment (annegentle) [inline labs/firepower-restapi-106/1.md]: "...certain types of objects," Seems like it should be plural objects.

File: labs/firepower-restapi-106/1.md [hunk 5]
```diff
@@ -27,37 +28,27 @@ which you cannot edit, with the exception of security zones and interface groups
...
+An object override allows you to define an alternate value for an object, which the system uses for the devices you specify.
+
+You can create an object whose definition works for most devices, and then use overrides to specify modifications to the object for the few devices that need different definitions. You can also create an object that needs to be overridden for all devices, but its use allows you to create a single policy for all devices. Object overrides allow you to create a smaller set of shared policies for use across devices, without giving up the ability to alter policies when needed for individual devices.
+
+***For example:*** <mark> You might want to deny ICMP traffic to the different departments in your company, each of which is connected to a different network. You can do this by defining an access control policy with a rule that includes a network object called Departmental Network. By allowing overrides for this object, you can then create overrides on each relevant device that specifies the actual network where that device is connected. </mark>
```
Comment (annegentle) [inline labs/firepower-restapi-106/1.md]: I'm not sure what the `<mark>` tags do here? Does it do something in the output?

**Review by annegentle — CHANGES_REQUESTED** [review]
Couple of notes inline.

**Comment by ReneeBornstein** [comment]
OMG, another weird error. I opened the GitHub doc link you sent, clicked on Files changed, and copied that to another window, so I could edit it. Usually I then just click on the pencil icon to edit the file. But this time I got this error:

[cid:image001.png@01D40BD1.FE73A850]

I’ve never had that happen before. I am logged in, so I guess somehow my push access (whatever that is) suddenly isn’t working?

**Review by ReneeBornstein — COMMENTED** [review]
Testing to see if I can edit after making this comment.

**Review by annegentle — APPROVED** [review]

**Pull request merged**